### DISCUSSION TITLE
Test scenario:  Creation of new FDS account

### DISCUSSION THREAD

**Discussion by crtahlin** [fairDataSociety/fdp-create-account#94] [Quality assurance]
FDP Create Account is available for registering FDS accounts and testing on Görli testnet as a web application.

We are looking for:
1. go through the procedure of registering an FDS account at 
https://create.dev.fairdatasociety.org/
2. login using the created account at
https://app.fairdrive.dev.fairdatasociety.org/

Please provide feedback as a thread in this issue and/or on [Fair Data Society Discord](https://discord.gg/KrVTmahcUA).

Currently known outstanding issues:
- [x] Chrome / Chromium browsers return an `Error while registering your account: Failed to construct 'Request': The duplex member must be specified for a request with a streaming body` in the process of registering. 
Issue is due to underlying `ky` libraries, tracked here: https://github.com/fairDataSociety/fdp-create-account/issues/77

**Comment by jbradach** [comment]
I've tried registering twice. It goes well at first, and I transfer gETH to the address, but then:

> Error while registering your account: Failed to construct 'Request': The `duplex` member must be specified for a request with a streaming body

Clicking try again just returns the same error and repeats a series of transactions.
- https://goerli.etherscan.io/address/0x82789CA4f5D3394EE1d1216411cC1404416C6024
- https://goerli.etherscan.io/address/0xfc6f71c0b6bea9cf7e18e1a5f63463783d2f14c3

**Comment by crtahlin** [comment]
> I've tried registering twice. It goes well at first, and I transfer gETH to the address, but then:
> 
> > Error while registering your account: Failed to construct 'Request': The `duplex` member must be specified for a request with a streaming body
> 
> Clicking try again just returns the same error and repeats a series of transactions.
> 
>     * https://goerli.etherscan.io/address/0x82789CA4f5D3394EE1d1216411cC1404416C6024
> 
>     * https://goerli.etherscan.io/address/0xfc6f71c0b6bea9cf7e18e1a5f63463783d2f14c3

@jbradach thank you for your feedback. 

It appears there is an issue with Chrome browser (and possibly others based on it) and the Creating FDS account app, due to underlying libraries. Hopefully they will be updated shortly or a workaround found. (Assuming you have used Chrome/Chromium.)

The app should not return the errors in Firefox browser. 

Will post an update here when updated.

**Comment by jbradach** [comment]
> 1. go through the procedure of registering an FDS account at
>    https://create.dev.fairdatasociety.org/

I was just able to successfully create an account using Firefox: 0x409aE6C3FAfe4dfB45cc11B8e9E00f1f1742d846.

> 2. login using the created account at
>    https://app.fairdrive.dev.fairdatasociety.org/

I also successfully logged in using the account I created.

Everything went smoothly this time!

**Comment by crtahlin** [comment]
The create account should now work in Firefox and Chrome.

**Comment by crtahlin** [comment]
Transformed the issue https://github.com/fairDataSociety/fdp-create-account/issues/48 into (this) discussion.
